hugo.yaml

```
@@ -27,10 +27,19 @@ params:
   # Icon pack "fab" includes social network icons, see: https://fontawesome.com/icons?d=gallery&s=brands&m=free
   # Icon pack "fas" includes solid style icons, see: https://fontawesome.com/icons?d=gallery&s=solid&m=free
   social:
-    - url: mailto:info@tina.pm
+    - url: https://github.com/Tina-pm
+      icon: github
+      icon_pack: fab
+    - url: https://mastodont.cat/@Tina@mastodon.ie
+      icon: mastodon
+      icon_pack: fab
+    - url: mailto:tina@tina.pm
```

Mastodon lo sacaría por ahora, IMHO. Y para mail mejor usemos info@tina.pm
```suggestion
    - url: mailto:info@tina.pm
```